src/cookbook/serverless-blog.md

```diff
@@ -0,0 +1,349 @@
+
+```html
+<template>
+    <div id="blog-home">
+        <h1>Blog</h1>
+        <!-- Create `v-for` and apply a `key` for Vue. -->
+        <div
+            v-for="post in posts"
+            :key="post.slug"
+        >
+        <router-link :to="{ name: 'BlogPost', params: { slug: post.slug } }">
+            <article class="media">
+            <figure>
+                <!-- Bind results using a `:` -->
+                <!-- Use a `v-if`/`else` if there is a `featured_image` -->
+                <img
+                    v-if="post.featured_image"
+                    :src="post.featured_image"
+                    alt=""
+                >
+                <img
+                    v-else
+                    src="http://via.placeholder.com/250x250"
+                    alt=""
+                >
+            </figure>
+            <h2>{{ post.title }}</h2>
+            <p>{{ post.summary }}</p>
+            </article>
+        </router-link>
+        </div>
+    </div>
+</template>
+
+<script>
+import { butter } from "@/buttercms"
+import { defineComponent, onMounted, ref } from "vue"
+import { RouterLink } from "vue-router"
+
+export default defineComponent({
+    name: "BlogHome",
+    components: {
+        RouterLink
+    },
+    setup () {
+        const posts = ref([])
+
+        onMounted( async () => {
+            posts.value = (
+                await butter.post.list({
+                    page: 1,
+                    page_size: 10,
+                })
+            ).data.data            
+        })
+
+        return {
+            posts,
+        }
+    },
+})
+</script>
+```
+
+::: info
+See the ButterCMS [API reference](https://buttercms.com/docs/api/?javascript#blog-posts) for additional options such as filtering by category or author. The response also includes some metadata that could be used for pagination.
+:::
+
+Here's what it looks like (note we added styling using [tailwind](https://tailwindcss.com/)):
+
+![buttercms-blog-list](/images/buttercms-blog-page-screenshot.png)
+
+Now create `views/BlogPost.vue` which will be your Blog Post page to display post details:
+
+```html
+<template>
+<div
+    id="blog-post"
+    v-if="loaded"
+>
+    <h1>{{ post.data.title }}</h1>
+    <h4>{{ post.data.author.first_name }} {{ post.data.author.last_name }}</h4>
+    <div v-html="post.data.body"></div>
+
+    <router-link
+        v-if="post.meta.previous_post"
+        :to="{ name: 'BlogPost', params: { slug: post.meta.previous_post.slug } }"
+    >
+        {{ post.meta.previous_post.title }}
+    </router-link>
+    <router-link
+        :to="{ name: 'BlogPost', params: { slug: post.meta.next_post.slug } }"
+        v-if="post.meta.next_post"
+    >
+        {{ post.meta.next_post.title }}
+    </router-link>
+</div>
+</template>
+
+<script>
+import { defineComponent, onMounted, ref, unref } from "vue"
+import { RouterLink, useRoute } from "vue-router"
+
+import { butter } from "@/buttercms.js"
```

Comment:
Lets be consistent here and ensure this is `import { butter } from "@/buttercms"`